cmd/influx/importer/v8.go

```diff
@@ -0,0 +1,222 @@
+package importer
+
+import (
+	"bufio"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"log"
+	"net/url"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/influxdb/influxdb/client"
+)
+
+const batchSize = 5000
+
+// V8Config is the config used to initialize a V8 importer
+type V8Config struct {
+	username, password string
+	url                url.URL
+	precision          string
+	writeConsistency   string
+	file, version      string
+	compressed         bool
+}
+
+// NewV8Config returns an initialized *V8Config
+func NewV8Config(username, password, precision, writeConsistency, file, version string, u url.URL, compressed bool) *V8Config {
+	return &V8Config{
+		username:         username,
+		password:         password,
+		precision:        precision,
+		writeConsistency: writeConsistency,
+		file:             file,
+		version:          version,
+		url:              u,
+		compressed:       compressed,
+	}
+}
+
+// V8 is the importer used for importing 0.8 data
+type V8 struct {
+	client                                     *client.Client
+	database                                   string
+	retentionPolicy                            string
+	config                                     *V8Config
+	wg                                         sync.WaitGroup
+	line, command                              chan string
+	done                                       chan struct{}
+	batch                                      []string
+	totalInserts, failedInserts, totalCommands int
+}
```

Comment:
The name `V8` doesn't tell what this thing is. It seems like all the code in this package should be moved to an `importer/v8` package. Otherwise it's going to get really crowded when we add other importers in the future. Also, the naming won't have `V8` duplicated everywhere and it'll sound better. e.g. `v8.Importer` instead of `importer.V8`.